src/main/java/appeng/core/sync/GuiBridge.java

```diff
@@ -358,24 +351,6 @@ public Object ConstructContainer( final InventoryPlayer inventory, final AEPartL
 
 			final Object o = target.newInstance( inventory, tE );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "o". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1488)